multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/auditlogging/model/AuditLogConfiguration.java

```diff
@@ -0,0 +1,81 @@
+package org.cloudfoundry.multiapps.controller.core.auditlogging.model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.cloudfoundry.multiapps.mta.model.AuditableConfiguration;
+import org.cloudfoundry.multiapps.mta.model.ConfigurationIdentifier;
+
+public class AuditLogConfiguration implements AuditableConfiguration {
+
+    private static String PERFORMED_ACTION_IDENTIFIER_KEY_NAME = "performed_action";
+    private static String TIME_IDENTIFIER_KEY_NAME = "time";
```

Comment:
What are we logging in the end?
I do not see username, spaceId, configuration, parameters used?
where is getConfigurationIdentifiers() method used?
I see only performedAction saved?

Reply:
They are used in the XSA project. Because the audit log happens there